lib/ansible/module_utils/facts/system/python.py

```diff
@@ -48,7 +49,20 @@ def collect(self, module=None, collected_facts=None):
             'executable': sys.executable,
             'has_sslcontext': HAS_SSLCONTEXT
         }
-
+        try:
+            site_package_path = site.getsitepackages()
+            python_facts['python'].update(
+                {'site_package_path': site_package_path}
+            )
+        except Exception:
+            pass
```

Comment:
it should show that we failed to get this value, not ignore and not set it at all

Reply:
@bcoca 
Should I return `{'site_package_path': None}` in case of an exception instead of ignoring it?

Reply:
I was thinking` 'N/A'` .. but` None `seems good enough to test against, we don't currently have a clear rule and many facts do different things.

Reply:
@bcoca 
Changed the result to return 'N/A' in case of exception. 

For me the 'unstable' `Test 2567 / 1 / 0 / 604` seems not to be related to this changes.

```
Failure: Assertion failed
				{
    "assertion": "ansible_mounts|default(\"UNDEF_MOUNT\") != \"UNDEF_MOUNT\" or ansible_distribution == \"MacOSX\"", 
    "changed": false, 
    "evaluated_to": false, 
    "msg": "Assertion failed"
}		
```